corehq/apps/hqwebapp/templates/hqwebapp/partials/form_list_form.html

```diff
@@ -25,7 +25,7 @@
         </thead>
 
         <tbody data-bind="foreach: rows">
-            <tr>
+            <tr data-bind="css: { error: Object.keys(errors).length !== 0 }">
```

Comment:
Usually better to use `_.keys` than `Object.keys` since underscore will polyfill for you if the browser doesn't have `Object.keys`